plugins/maximize/plugin.js

```diff
@@ -113,24 +120,62 @@
 			var outerScroll;
 
 			// Saved resize handler function.
-			function resizeHandler() {
-				var viewPaneSize = mainWindow.getViewPaneSize();
-				editor.resize( viewPaneSize.width, viewPaneSize.height, null, true );
+			function resizeHandler( height ) {
+				var viewPaneSize = mainWindow.getViewPaneSize(),
+					panelHeight = checkIOS() && height ? height : viewPaneSize.height;
+
+				editor.resize( viewPaneSize.width, panelHeight, null, true );
+			}
+
+			function setContainerHeight( container ) {
```

Comment:
Again, this function changes container height but apart from that does some magic with scroll position so its name is only partially relevant here.

From what I see it is used only once in the code, so maybe could be inlined into `focus` handler?